benchmark/blockmap.js

```diff
@@ -0,0 +1,20 @@
+var path = require( 'path' )
+var fs = require( 'fs' )
+var bench = require( 'nanobench' )
+var BlockMap = require( '..' )
+
+const ITERATIONS = 1000
```

Comment:
Nitpick: why is only `ITERATIONS` a `const`? Everything except `blockMap` and `i` could be a `const` here.